deploy/resources/v0.8.0/addons/pipelinetemplates/s2i-go/s2i-gov0-8-0-build-deploy.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: tekton.dev/v1alpha1
```

Comment:
* could you please name the dir `pipelines` 
* and follow the same convention used in clustertasks ?
* why do we need a versioned pipeline?

Reply:
Changed to pipelines,
After testing with the console, it makes sense to have non-versioned pipeline only.